index.md

```diff
@@ -1,12 +1,11 @@
 ---
-# You don't need to edit this file, it's empty on purpose.
-# Edit theme's home layout instead if you wanna make some changes
-# See: https://jekyllrb.com/docs/themes/#overriding-theme-defaults
 layout: home
 ---
 
 Turku &hearts; Frontend is the #1 community for frontend developers and designers in Turku.
 
 We organize monthly meetups, hackathons, programming competitions and other gatherings to promote developer culture. Our doors are open for everyone: beginners, curious, hobbyists, professionals and beyond.
 
-Our meetups would not be possible without our [amazing speakers](/speakers)
+Our meetups would not be possible without our [amazing speakers](/speakers).
+
+Please note that everyone on our meetups is expected act in accordance with the [Code of Conduct](/code-of-conduct) so we can build a community that is open and welcoming to everyone.
```

Comment:
> Please note that everyone on our meetups is expected act in accordance with

Still typos here. I'd use "in" instead of "on" though I'm not 100% sure which is more correct here.

But in any case there's at least a "to" missing from "is expected _to_ act"